flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
Mind patching the appdata here instead of skipping the check?

Reply:
I'd prefer not to touch upstream source whenever possible.